packages/cli/src/commands/setup/deploy/providers/flightcontrol.js

```diff
@@ -0,0 +1,211 @@
+
+  if (detectedDatabase === database) {
+    let dbService
+    switch (database) {
+      case 'postgresql':
+        dbService = postgresDatabaseService
+        break
+      case 'mysql':
+        dbService = mysqlDatabaseService
+        break
+      default:
+        throw new Error(`
+       Unexpected datasource provider found: ${database}`)
+    }
+    return {
+      path: path.join(getPaths().base, 'flightcontrol.json'),
+      content: {
+        ...flightcontrolConfig,
+        environments: [
+          {
+            ...flightcontrolConfig.environments[0],
+            services: [
+              ...flightcontrolConfig.environments[0].services.map((service) => {
+                if (service.id === 'redwood-api') {
+                  return {
+                    ...service,
+                    envVariables: databaseEnvVariables,
+                  }
+                }
+                return service
+              }),
+              dbService,
+            ],
+          },
+        ],
+      },
+    }
+  } else {
+    throw new Error(`
+    Prisma datasource provider is detected to be ${detectedDatabase}.
+
+    Update your schema.prisma provider to be postgresql or mysql, then run
+    yarn rw prisma migrate dev
+    yarn rw setup deploy flightcontrol
+    `)
+  }
+}
+
+const updateGraphQLFunction = () => {
+  return {
+    title: 'Adding CORS config to createGraphQLHandler...',
+    task: (_ctx) => {
+      const graphqlTsPath = path.join(
+        getPaths().base,
+        'api/src/functions/graphql.ts'
+      )
+      const graphqlJsPath = path.join(
+        getPaths().base,
+        'api/src/functions/graphql.js'
+      )
+
+      let graphqlFunctionsPath
+      if (fs.existsSync(graphqlTsPath)) {
+        graphqlFunctionsPath = graphqlTsPath
+      } else if (fs.existsSync(graphqlJsPath)) {
+        graphqlFunctionsPath = graphqlJsPath
+      } else {
+        console.log(`
+    Couldn't find graphql handler in api/src/functions/graphql.js.
+    You'll have to add the following cors config manually:
+
+      cors: { origin: '*', credentials: true}
+    `)
+        return
+      }
+
+      const graphqlContent = fs
+        .readFileSync(graphqlFunctionsPath, 'utf8')
+        .split(EOL)
+      const graphqlHanderIndex = graphqlContent.findIndex((line) =>
+        line.includes('createGraphQLHandler({')
+      )
+
+      if (graphqlHanderIndex === -1) {
+        console.log(`
+    Couldn't find graphql handler in api/src/functions/graphql.js.
+    You'll have to add the following cors config manually:
+
+      cors: { origin: '*', credentials: true}
+    `)
+        return
+      }
+
+      graphqlContent.splice(
+        graphqlHanderIndex + 1,
+        0,
+        "  cors: { origin: '*', credentials: true },"
+      )
+
+      fs.writeFileSync(graphqlFunctionsPath, graphqlContent.join(EOL))
+    },
+  }
+}
+
+// We need to set the apiUrl evn var for local dev
+const addToDotEnvDefaultTask = () => {
```

Comment:
Because the apiUrl was changed to use an env var, we need to make sure there's a value for local dev